refactor: fix middleware URL generation (#6688)
vuestorefront/vue-storefront: packages/core/docs/reference/migrate/2.5.11/overview.md

# Migrating projects to 2.5.11

## Update `nuxt.config.js`

> In this release, we've added the `middlewareUrl` and `ssrMiddlewareUrl` optional properties. You should use them to avoid security vulnerability.
> To enable this properties open the  `nuxt.config.js` file and add the `middlewareUrl`  and `ssrMiddlewareUrl` property in the `publicRuntimeConfig` object like shown below:

Reviewer: ```suggestion
In this release, we've added the `middlewareUrl` and `ssrMiddlewareUrl` optional properties. You should use them to avoid security vulnerability.
To enable these properties, open the  `nuxt.config.js` file and add the `middlewareUrl`  and `ssrMiddlewareUrl` properties to the `publicRuntimeConfig` object as shown below:
```

:::warning

> Make sure to pass whole url with protocol and/or port and suffix it with `/api/`.

Reviewer: ```suggestion
Make sure to pass the whole URL with protocol, port (if applicable), and suffix it with `/api/`.
```
